bin/node/cli/src/chain_spec.rs

```diff
@@ -550,5 +551,6 @@ fn pangolin_development_genesis() -> pangolin_runtime::GenesisConfig {
 		},
 		darwinia_evm: pangolin_runtime::EVMConfig { accounts: evm_accounts },
 		dvm_ethereum: Default::default(),
+		darwinia_bridge_ethereum_bsc: Default::default(),
```

Comment:
What genesis header should be used here?

Reply:
lastest finalized checkpoint header. relayer and this pallet should start with the same header number